fundamentos-do-desenvolvimento-web/bloco-03-int-html-css/dia-01/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+  <head>
+    <meta charset="UTF-8">
+    <title>HTML</title>
```

Comment:
`suggestion | nitpick`

talvez um título mais descritivo poderia melhorar o entendimento da página

```suggestion
    <title>Amizades da Trybe</title>
```